{"Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test": ["tensorflow.python.platform.test.main"], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest._CountReferences": ["gc.get_objects", "<builtin>.isinstance"], "gc.get_objects": [], "<builtin>.isinstance": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testUniqueClassForEachErrorCode": ["tensorflow.python.platform.test.TestCase.assertTrue", "<builtin>.isinstance", "tensorflow.python.platform.test.TestCase.assertEqual", "tensorflow.python.framework.errors_impl._make_specific_exception", "tensorflow.python.framework.errors_impl.error_code_from_exception_type"], "tensorflow.python.framework.errors_impl._make_specific_exception": [], "tensorflow.python.platform.test.TestCase.assertTrue": [], "tensorflow.python.framework.errors_impl.error_code_from_exception_type": [], "tensorflow.python.platform.test.TestCase.assertEqual": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testKnownErrorClassForEachErrorCodeInProto": ["tensorflow.python.platform.test.TestCase.assertTrue", "<builtin>.isinstance", "tensorflow.python.platform.test.TestCase.assertEqual", "<builtin>.len", "tensorflow.python.framework.errors_impl._make_specific_exception", "tensorflow.core.lib.core.error_codes_pb2.Code.values", "warnings.catch_warnings"], "tensorflow.core.lib.core.error_codes_pb2.Code.values": [], "warnings.catch_warnings": [], "<builtin>.len": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testUnknownErrorCodeCausesWarning": ["tensorflow.python.platform.test.TestCase.assertTrue", "<builtin>.str", "<builtin>.isinstance", "tensorflow.python.platform.test.TestCase.assertEqual", "<builtin>.len", "tensorflow.python.framework.errors_impl._make_specific_exception", "warnings.catch_warnings", "tensorflow.python.framework.errors_impl.error_code_from_exception_type"], "<builtin>.str": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testStatusDoesNotLeak": ["tensorflow.python.util.compat.as_bytes", "tensorflow.python.lib.io._pywrap_file_io.DeleteFile", "tensorflow.python.platform.test.TestCase.assertEqual", "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest._CountReferences", "gc.collect"], "tensorflow.python.util.compat.as_bytes": [], "tensorflow.python.lib.io._pywrap_file_io.DeleteFile": [], "gc.collect": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testPickleable": ["pickle.loads", "tensorflow.python.platform.test.TestCase.assertEqual", "tensorflow.python.framework.errors_impl._make_specific_exception", "pickle.dumps"], "pickle.dumps": [], "pickle.loads": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testErrorPayloadsFromStatus": ["tensorflow.python.platform.test.TestCase.assertEqual", "tensorflow.python.framework._errors_test_helper.TestRaiseFromStatus", "tensorflow.python.platform.test.TestCase.assertRaises"], "tensorflow.python.platform.test.TestCase.assertRaises": [], "tensorflow.python.framework._errors_test_helper.TestRaiseFromStatus": [], "Desktop\\Work\\tensorflow-master\\tensorflow\\python\\framework\\errors_test.ErrorsTest.testErrorPayloadsDefaultValue": ["tensorflow.python.platform.test.TestCase.assertEqual", "<builtin>.type", "<builtin>.len"], "<builtin>.type": [], "tensorflow.python.platform.test.main": []}
